tests/jballerina-integration-test/src/test/resources/configurables/configurableProject/main/main.bal

```diff
@@ -29,24 +28,83 @@ configurable string[] & readonly stringArr = ["apple", "orange", "banana"];
 configurable boolean[] & readonly booleanArr = [true, true];
 configurable decimal[] & readonly decimalArr = ?;
 
+type AuthInfo record {|
+    readonly string username;
+    int id = 100;
+    string password;
+    string[] scopes?;
+    boolean isAdmin = false;
+|};
+
+type UserTable table<AuthInfo> key(username);
+
+configurable AuthInfo & readonly admin = ?;
+configurable UserTable & readonly users = ?;
+
 public function main() {
+    testSimpleValues();
+    testArrayValues();
+    testRecordValues();
+    testTableValues();
+
+    print("Tests passed");
```

Comment:
I think these print statements are unnecessary, because the `assertEquals` methods will fail the tests if the assertion fails. In this case we can get rid of the log leechers as well.

Reply:
As these are integration tests, the assertion errors are logged and not failing the tests if an assertion fails. 
So the print statements are needed for the check. 
![test](https://user-images.githubusercontent.com/28644893/107319280-7f7d8980-6ac4-11eb-8913-9e8a49206525.png)

Reply:
Then let's have these